ick/cmdline.py

```diff
@@ -186,6 +186,7 @@ def run(
                 else:
                     path.parent.mkdir(parents=True, exist_ok=True)
                     path.write_bytes(mod.new_bytes)
+                print(f"   Change made: {mod.filename:30s} {mod.diffstat}")
```

Comment:
Slightly unrelated change, non-dry-run was hard to figure out what the _reasons_ were that files changed.